build.gradle

```diff
@@ -53,7 +53,11 @@ task requireJavadoc(type: JavaExec) {
 
 task expectedTestOutputsMustCompile(type: Exec) {
     // TODO: should this task run in CI, or as part of the regular tests?
-    commandLine "sh", "typecheck_test_outputs.sh"
+    if (System.getProperty('os.name').toLowerCase().startsWith('windows')) {
+        commandLine "cmd", "/c", "your_batch_file.bat"
```

Comment:
I think this should be `typecheck_test_outputs.bat` rather than `your_batch_file.bat`.

Have you tried running this task and checking to make sure it does what you expect? I would think that this problem would show up immediately if you did.

Reply:
Yes, I did, sorry. I accidentally deleted the file before I committed so I had to re-copy and paste the template code, but it looks like I forgot to change it again, thanks!